libraries/AP_AHRS/AP_AHRS.h

```diff
@@ -188,7 +188,8 @@ class AP_AHRS : public AP_AHRS_DCM {
     void writeExtNavVelData(const Vector3f &vel, float err, uint32_t timeStamp_ms, uint16_t delay_ms) override;
 
     // get speed limit
-    void getEkfControlLimits(float &ekfGndSpdLimit, float &ekfNavVelGainScaler) const;
+    void getEkfControlLimits(float &ekfGndSpdLimit, float &controlScaleXY) const;
```

Comment:
```suggestion
    void getControlLimits(float &gndSpdLimit, float &controlScaleXY) const;
```

I'd like to split this call at some stage.  Several callers only want one of these.

Reply:
done